shell/src/main/java/org/apache/accumulo/shell/commands/GrepCommand.java

```diff
@@ -106,7 +113,9 @@ public String description() {
   public Options getOptions() {
     final Options opts = super.getOptions();
     numThreadsOpt = new Option("nt", "num-threads", true, "number of threads to use");
+    negateOpt = new Option("v", "negate", false, "only include rows without search term");
```

Comment:
`-v` is typically reserved for `--verbose`; I think it could be confusing to use `-v` for this option.

Reply:
Hmm, that's a good point. Okay, you've convinced me.